CMakeLists.txt

```diff
@@ -42,7 +51,7 @@ option(USE_EXTERNAL_LCMS "Use system install lcms2 library." OFF)
 # Versioning
 
 if(NOT SOVERSION)
-	set(SOVERSION "${OpenColorIO_VERSION_MAJOR}_${OpenColorIO_VERSION_MINOR}" CACHE STRING "Set the SO version in the SO name of the output library")
+	set(SOVERSION "${OpenColorIO_VERSION_MAJOR}_${OpenColorIO_VERSION_MINOR}${OpenColorIO_VERSION_RELEASE_TYPE}" CACHE STRING "Set the SO version in the SO name of the output library")
```

Comment:
Do we really need to bake the "dev" flag into the SO / namespace for dev builds?

Reply:
The dev using the library must *see*  that it's a work-in-progress.